Specs/Scene/PerspectiveFrustumSpec.js

```diff
@@ -145,6 +145,26 @@ defineSuite([
         expect(pixelSize.y).toEqual(expected.y);
     });
 
+    it('get pixel dimensions throws without canvas width', function() {
```

Comment:
Remove these two exception tests (since they are covered by the off center test) or include all the exception tests here.